luthier/src/main/java/com/yahoo/bard/webservice/config/luthier/factories/SingleDataSourcePhysicalTableFactory.java

```diff
@@ -0,0 +1,94 @@
+// Copyright 2019 Oath Inc.
+// Licensed under the terms of the Apache license. Please see LICENSE.md file distributed with this work for terms.
+package com.yahoo.bard.webservice.config.luthier.factories;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import com.yahoo.bard.webservice.config.luthier.Factory;
+import com.yahoo.bard.webservice.config.luthier.LuthierIndustrialPark;
+import com.yahoo.bard.webservice.config.luthier.LuthierValidationUtils;
+import com.yahoo.bard.webservice.data.config.LuthierPhysicalTableParams;
+import com.yahoo.bard.webservice.data.config.LuthierTableName;
+import com.yahoo.bard.webservice.data.dimension.DimensionColumn;
+import com.yahoo.bard.webservice.data.metric.MetricColumn;
+import com.yahoo.bard.webservice.data.time.DefaultTimeGrain;
+import com.yahoo.bard.webservice.table.ConfigPhysicalTable;
+
+import org.joda.time.DateTimeZone;
+
+import java.util.LinkedHashMap;
+import java.util.LinkedHashSet;
+import java.util.Locale;
+
+/**
+ * A factory that is used by default to support Simple (non-Composite) Physical Table.
+ * Classes that extend this abstract class:
```

Comment:
Specifying this isn't necessary. If JavaDoc is generated, the JavaDoc will figure this out for you, and
if people have access to the codebase, they'll have some strategy for using their tools to find the subclasses.
Probably they'll use their IDE.

Reply:
Makes sense. I will remove that then.